### PR TITLE
Schedule document augmentation

### PR DESCRIPTION
**Pull request by tulitheprogrammer**
Added new "customized" flag into schedule time slot structure
Purpose:  
Differentiating student's customized scheduled lessons (time slots) from class.
This is a mandatory fix for portal (FE) issue fix
 https://github.com/myspecialway/my-special-way-portal/issues/258

### CODE REVIEW

**Comment by tulitheprogrammer** [comment]
Hi @dimkinv, 
Can this PR be merged into master, please?

File: src/models/timeslot.db.model.ts [hunk 1]
```diff
@@ -6,4 +6,5 @@ export interface TimeSlotDbModel {
     index: string;
     lesson?: LessonDbModel;
     room?: LocationDbModel;
+    customized?: boolean;
```
Comment (dimkinv) [inline src/models/timeslot.db.model.ts]: Can you please elaborate again why do we need it? The initial idea was that if timeslot is on class belongs to class and if on student it is the exception on the class timeslot.

Comment (tulitheprogrammer) [reply src/models/timeslot.db.model.ts]: Yes sure, @dimkinv.
 without this flag you can't tell the difference between the exception and the class's time slot.
As for today, you get a merged schedule from server and return only one possible exception per student.

File: src/modules/graphql/schemas/timeslot.gql [hunk 2]
```diff
@@ -1,13 +1,15 @@
 # A type describing a single time slot on the class/student schedule
 type TimeSlot {
-    # 2 numeric chars (00, 01...) represents the [hour, day] of timeslot 
+    # 2 numeric chars (00, 01...) represents the [hour, day] of timeslot
```
Comment (dimkinv) [inline src/modules/graphql/schemas/timeslot.gql]: These are now `0_0` separated by underscore.
I'm approving this PR, but can you please update this comment just before you merge it?

Comment (tulitheprogrammer) [reply src/modules/graphql/schemas/timeslot.gql]: Hi @dimkinv ,
Done

**Pull request merged**